style.css

```
@@ -0,0 +1,131 @@
+:root {
+  --font-family-heading: "Anton", sans-serif;
+  --font-family-body: "Noto Sans JP", sans-serif;
+
+  --color-white: #fffefc;
+  --color-black: #020911;
+  --color-accent-yellow: #f2e52e;
+  --color-accent-red: #f20574;
+  --color-background-black: #171525;
+  --color-background-navy: #030427;
+  --color-form-background-yellow: #fffab9;
+  --color-form-background-qrav: #ad9783;
```

こちらも細かいですが正しくはgrayです
```suggestion
  --color-form-background-gray: #ad9783;
```